src/model/geometry.cpp

```diff
@@ -46,8 +43,7 @@ void Geometry::populate_buffers()
     glBindVertexArray(0);
 }
 
-void Geometry::attachNewTexture(const char *texture_loc)
-{
+void Geometry::attachNewTexture(const char *texture_loc) {
```

Comment:
This function loads texture assets from disk by filename. Should be put in asset loader, and some way to allow systems to use loaded textures.